build.pl

```diff
@@ -195,6 +196,15 @@
 {
 	$monoHostArch = $arch32 ? "i686" : "x86_64";
 	$existingExternalMono = "$existingExternalMonoRoot/linux";
+
+	#The build is running through windows subsystem for linux. Here we can build the classlibs
+	#But at the moment there are issues building the runtime that need more time to work through
+	if (`which notepad.exe` ne "")
```

Comment:
I'm thinking I'll change this to instead be passed down by a new script build_classlibs_wsl.pl rather than running build_classlibs_osx and then autodetecting wsl on windows